### PR TITLE
Resolved 3 critical errors

### PR DESCRIPTION
**Pull request by hortinstein**
This fixes 3 packages within CENTOS that needed to be updated.  Errors can be seen here.

https://quay.io/repository/hortinstein/nessus/image/306f1598f7790560dbf18f828b462c0dd5e761b180b3362d3aad889b95974ba3?tab=vulnerabilities

### CODE REVIEW

**Comment by jumanjiman** [comment]
[Clair](https://github.com/coreos/clair) FTW!

File: Dockerfile [hunk 1]
```diff
@@ -11,10 +11,16 @@ MAINTAINER Tim Kropp <timkropp77@gmail.com>
 # Update the base image.
 # RUN yum -y update
 
+RUN yum -y update nss-util
+RUN yum -y update bind-license
+RUN yum -y update libssh2
```
Comment (jumanjiman) [inline Dockerfile]: it's usually better to update all three in one run, followed by `yum clean all` to keep the image layer small.

example:

```
RUN yum -y update \
      nss-util \
      bind-license \
      libssh2 \
    && yum clean all
```

File: Dockerfile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # http://tenable.com
 # https://index.docker.io/_/centos/
-FROM centos
+FROM centos:latest
```
Comment (jumanjiman) [inline Dockerfile]: nit: this should really be something like `centos:7` in an ideal case

**Comment by sometheycallme** [comment]
>Clair FTW!

Indeed

@hortinstein - thanks for bringing attention to this problem.   We spend a little cycle time on maintaining, and any help is appreciated.

@jumanjiman is right, if we concatenate the yum commands into a single line it should work.

There was also a problem with the Docker version we are using for Circle so I took a stab at repairing both of these here:

https://github.com/cleanerbot/docker-nessus/pull/25

**Comment by sometheycallme** [comment]
Closing this PR in favor of https://github.com/cleanerbot/docker-nessus/pull/25

kindly rebase.

**Pull request closed without merging**